### PR TITLE
ath79: add support for Senao Engenius EAP1750H

### PR DESCRIPTION
**Pull request by mcprat**
this board is equivalent hardware to Engenius EAP1200H

first, change EAP1200H wifi calibration from script to nvmem-cells

rebased right before commit a01d23e755ba46f41e667d558d82d4871d7f5450
because of issue #11319

### CODE REVIEW

File: target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata [hunk 1]
```diff
@@ -14,6 +14,7 @@ case "$FIRMWARE" in
 	araknis,an-500-ap-i-ac|\
 	araknis,an-700-ap-i-ac|\
 	engenius,eap1200h|\
+	engenius,eap1750h|\
 	engenius,enstationac-v1|\
```
Comment (DragonBluep) [inline target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata:18]: It's better to convert calibration data to nvmem-cells. https://github.com/openwrt/openwrt/commit/48625a04453b4f5775db2d2a5abebf977b81344c

Comment (mpratt14) [reply target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata]: thanks for reminding me about that new method, it works

**Comment by hauke** [comment]
Thank you for the patches, I applied them to master in 128947db42036bcc98fa7f6d70e3b2e5cb494e18 and 6de9287abd699de5645dc50ce88eefbb60298e7b.

**Comment by mpratt14** [comment]
@hauke thanks for the quick merge. did you happen to look at the issue I mentioned in the top post here? I believe it's a very simple fix and my patch linked there takes care of it. As of right now, snapshot images for this board and similar boards using the lzma-loader the same way (and 22.03.2 images for similar boards) causes soft-bricking still.

**Comment by hauke** [comment]
I had a quick look at the patch, but I need some more time to understand that part of the code.

**Pull request closed without merging**